src/config/units.yaml

```diff
@@ -18,43 +18,116 @@ duration:
   description: refers to the duration of the observation
   unit: seconds
   aggregation: sum
-timestamp:
-  description: refers to the time of occurrence of the observation
-  unit: RFC3339
-  aggregation: None
-carbon:
-  description: an amount of carbon emitted into the atmosphere
-  unit: gCO2e
-  aggregation: sum
-energy:
-  description: amount of energy utilised by the component
-  unit: kWh
-  aggregation: sum
 e-cpu:
   description: Energy consumed by the CPU of the component
   unit: kWh
   aggregation: sum
+el:
+  description: expected lifespan for some component
+  unit: seconds
+  aggregation: none
 e-mem:
   description: Energy consumed by the Memory of the component
   unit: kWh
   aggregation: sum
-e-net:
-  description: Energy consumed by the Network of the component
-  unit: kWh
-  aggregation: sum
 embodied-carbon:
   description: Embodied Emissions of the component
   unit: gCO2e
   aggregation: sum
-operational-carbon:
-  description: Operational Emissions of the component
-  unit: gCO2e
+energy:
+  description: amount of energy utilised by the component
+  unit: kWh
+  aggregation: sum
+e-net:
```

Comment:
```suggestion
energy-net:
```

Reply:
using `energy-network` for consistency with `energy-memory`